### PR TITLE
Example code for STL

### PR DESCRIPTION
**Pull request by astrophenomenon**
Example code for vector and map

### CODE REVIEW

**Review by BMPark-Lab — APPROVED** [review]
Looks good~

**Review by hmlearning — APPROVED** [review]
good job!

File: cpp1st/week03/jongsang/Person.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#include "Person.h"
+#include <iostream>
+
+// defautl constructor
```
Comment (YonghoKang) [inline cpp1st/week03/jongsang/Person.cpp:4]: Typo

File: cpp1st/week03/jongsang/Person.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#include "Person.h"
+#include <iostream>
+
+// defautl constructor
+Person::Person()
+{
+    std::wcout << L"Person default constructor" << std::endl;
+}
+// consturctor
```
Comment (YonghoKang) [inline cpp1st/week03/jongsang/Person.cpp:9]: Typo

File: cpp1st/week03/jongsang/Person.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+// defautl constructor
+Person::Person()
+{
+    std::wcout << L"Person default constructor" << std::endl;
+}
+// consturctor
+Person::Person(std::wstring _name, int _age) : name(_name), age(_age)
+{
+    std::wcout << L"Person consturctor" << std::endl;
```
Comment (YonghoKang) [inline cpp1st/week03/jongsang/Person.cpp:12]: Typo

**Review by th4inquiry — APPROVED** [review]
Looks good!

File: cpp1st/week03/jongsang/VectorTest.cpp [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+
+void VectorTest::testIterator()
+{
+    std::wcout << L"\n====" << __FUNCTIONW__ << L"====" << std::endl;
+
+    // Normally, iterator is faster than indexing except Vector
+
+    // Iteraotr for int vector
```
Comment (YonghoKang) [inline cpp1st/week03/jongsang/VectorTest.cpp:36]: Typo

**Review by YonghoKang — APPROVED** [review]
Looks good. Fix some typo before merging

File: cpp1st/week03/jongsang/Person.cpp (hunk 1 above)
Comment (astrophenomenon) [reply cpp1st/week03/jongsang/Person.cpp]: I have changed it. Thanks

File: cpp1st/week03/jongsang/Person.cpp (hunk 2 above)
Comment (astrophenomenon) [reply cpp1st/week03/jongsang/Person.cpp:9]: I will change it. Thanks

File: cpp1st/week03/jongsang/Person.cpp (hunk 3 above)
Comment (astrophenomenon) [reply cpp1st/week03/jongsang/Person.cpp:12]: I will change it.

File: cpp1st/week03/jongsang/VectorTest.cpp (hunk 4 above)
Comment (astrophenomenon) [reply cpp1st/week03/jongsang/VectorTest.cpp]: I will change it.

**Pull request merged**